src/legacy/areas/picker/_picker.scss

```
@@ -91,6 +91,60 @@
     .tools-field-title .input-text {
       font-weight: var(--number-font-weight-bold);
     }
+
+    .field-textarea + .field-textarea {
+      margin-top: 0;
+    }
+
+    .field-textarea {
+      height: 0;
+      overflow: hidden;
+
+      &.field-textarea--selected {
+        height: auto;
+      }
+    }
+
+    .textarea-navigation {
+      ul {
+        display: flex;
+        list-style: none;
+        margin: 0;
+        padding: var(--size-spacing-half-again);
+        padding-bottom: 0;
+
+        @media screen and (min-width: tokens.$size-breakpoint-md) {
+          padding-left: var(--size-spacing-double);
+        }
+      }
+
+      li + li {
+        @media screen and (min-width: tokens.$size-breakpoint-md) {
+          margin-left: var(--size-spacing-half-again);
+        }
+      }
+
+      .tab {
+        border-top-left-radius: var(--size-border-radius-default);
+        border-top-right-radius: var(--size-border-radius-default);
+        cursor: pointer;
+        display: block;
+        font-size: 0.875rem;
+        padding: var(--size-spacing-default);
+      }
+
+      @media (hover) {
+        .tab:hover,
+        .tab:focus {
+          background-color: var(--color-form-bg);
+        }
+      }
```

I've never seen hover styles wrapped in a media query like this. (Apologies if you copied this from elsewhere in the MLTSHP codebase).

My concern is this locks the focus styles behind the hover media query, so keyboard users wouldn't see it.

Unless I'm missing something, I think it'd be better to remove the media query. The hover styles won't hurt anything for users in devices that don't support hover.

```suggestion
      .tab:hover,
      .tab:focus {
        background-color: var(--color-form-bg);
      }
```

---

Ah yes, the reason for this is to avoid two interactions on touch devices—where the first tap shows the hover effect the second tap serves as a click. I'll test this out on mobile and navigating with a keyboard using VoiceOver and make sure it's reasonable on each.

---

Yeah, seems like the media query is unnecessary. Thanks for the review!